test/test.ini

```diff
@@ -30,6 +30,9 @@ authdb = port=6666 host=127.0.0.1 dbname=p1 auth_user=pswcheck
 
 hostlist1 = port=6666 host=127.0.0.1,::1 dbname=p0 user=bouncer
 hostlist2 = port=6666 host=127.0.0.1,127.0.0.1 dbname=p0 user=bouncer
+hostlist_good_first = port=6666 host=127.0.0.1,127.0.0.3 dbname=p0 user=bouncer load_balance_hosts=disable
+hostlist_bad_first = port=6666 host=unresolvable-hostname,127.0.0.1 dbname=p0 user=bouncer load_balance_hosts=disable
+load_balance_hosts_update = port=6666 host=127.0.0.1,127.0.0.3 dbname=p0 user=bouncer load_balance_hosts=disable
```

Comment:
The load_balance_hosts_update database isn't used (afaict). Did you have a test in mind for it?